### ISSUE TITLE
Unhandled exception in RatScanner 3.0.0

### ISSUE THREAD

**Issue by Spazjuaner** [RatScanner/RatScanner#165]
**Error**
Unhandled exception in RatScanner 3.0.0
```
System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation.
 ---> System.NullReferenceException: Object reference not set to an instance of an object.
   at RatScanner.ViewModel.MainWindowVM.get_IconLink()
   at RatRazor.Pages.App.Index.<BuildRenderTree>b__0_1(RenderTreeBuilder __builder3)
   at Microsoft.AspNetCore.Components.Rendering.RenderTreeBuilder.AddContent(Int32 sequence, RenderFragment fragment)
   at MudBlazor.MudItem.BuildRenderTree(RenderTreeBuilder __builder)
   at Microsoft.AspNetCore.Components.ComponentBase.<.ctor>b__6_0(RenderTreeBuilder builder)
   at Microsoft.AspNetCore.Components.Rendering.ComponentState.RenderIntoBatch(RenderBatchBuilder batchBuilder, RenderFragment renderFragment, Exception& renderFragmentException)
--- End of stack trace from previous location ---
   at Microsoft.AspNetCore.Components.WebView.Services.WebViewRenderer.HandleException(Exception exception)
   at Microsoft.AspNetCore.Components.R
```
<details>
<summary>Log</summary>

```
[20:04:59.5399369] > [Info]  ----- RatScanner 3.0.0 -----
[20:04:59.5400854] > [Info]  Starting RatScanner...
[20:04:59.5400886] > [Info]  Prewarming name tool tip...
[20:04:59.5552220] > [Info]  Prewarming icon tool tip...
[20:04:59.5713271] > [Info]  Checking for new updates...
[20:04:59.5771636] > [Info]  Loading resource "RSClientVersion"...
[20:05:00.6395918] > [Info]  Loading config...
[20:05:00.6453989] > [Info]  Loading item data...
[20:05:00.6455643] > [Info]  Loading resource "RSItemDataVersion"...
[20:05:02.3390464] > [Info]  Loading TarkovTracker...
[20:05:03.5632762] > [Error] Unhandled exception in RatScanner 3.0.0
 -------------------- 
 System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation.
 ---> Newtonsoft.Json.JsonReaderException: Unexpected character encountered while parsing value: E. Path '', line 0, position 0.
   at Newtonsoft.Json.JsonTextReader.ParseValue()
   at Newtonsoft.Json.JsonReader.ReadForType(JsonContract contract, Boolean hasConverter)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.Deserialize(JsonReader reader, Type objectType, Boolean checkAdditionalContent)
   at Newtonsoft.Json.JsonSerializer.DeserializeInternal(JsonReader reader, Type objectType)
   at Newtonsoft.Json.JsonConvert.DeserializeObject(String value, Type type, JsonSerializerSettings settings)
   at Newtonsoft.Json.JsonConvert.DeserializeObject[T](String value, JsonSerializerSettings settings)
   at Newtonsoft.Json.JsonConvert.DeserializeObject[T](String value)
   at RatTracking.TarkovTrackerDB.UpdateToken() in D:\Programmierung\Projects\RatScanner\RatScanner\RatTracking\TarkovTrackerDB.cs:line 60
   at RatTracking.TarkovTrackerDB.ValidToken() in D:\Programmierung\Projects\RatScanner\RatScanner\RatTracking\TarkovTrackerDB.cs:line 42
   at RatTracking.TarkovTrackerDB.Init() in D:\Programmierung\Projects\RatScanner\RatScanner\RatTracking\TarkovTrackerDB.cs:line 21
   at RatScanner.RatScannerMain..ctor()
   at RatScanner.RatScannerMain.get_Instance()
   at RatScanner.View.BlazorUI.<>c.<.ctor>b__8_0(IServiceProvider s)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteVisitor`2.VisitCallSiteMain(ServiceCallSite callSite, TArgument argument)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.VisitRootCache(ServiceCallSite callSite, RuntimeResolverContext context)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteVisitor`2.VisitCallSite(ServiceCallSite callSite, TArgument argument)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.Resolve(ServiceCallSite callSite, ServiceProviderEngineScope scope)
   at Microsoft.Extensions.DependencyInjection.ServiceProvider.CreateServiceAccessor(Type serviceType)
   at System.Collections.Concurrent.ConcurrentDictionary`2.GetOrAdd(TKey key, Func`2 valueFactory)
   at Microsoft.Extensions.DependencyInjection.ServiceProvider.GetS
```
</details>

**Comment by Spazjuaner** [comment]
Oh seems BSG has just released a small patch, this is most likely why we are getting this error.

**Comment by Blightbuster** [comment]
Fixed in [v3.1.0](https://github.com/RatScanner/RatScanner/releases/tag/v3.1.0).
Please comment if this issue persists and I will reopen it.

**Comment by Spazjuaner** [comment]
Thank you! <3
